src/style.css

```diff
@@ -0,0 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: aliceblue;
+}
+
+.main-body {
+  height: 100vh;
+  width: 60vw;
+  margin: auto;
+  border-radius: 20px;
+  box-shadow: 0 0 3px gray;
+  padding-top: 1vh;
+  background-color: transparent;
+}
+
+h1 {
+  margin-left: 25vw;
+  margin-top: 10vh;
+}
+
+.recent-scores {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 20vw;
+}
+
+.refresh {
+  width: 5vw;
+  height: 4vh;
+  border-radius: 5px;
+  background-color: blue;
+  color: white;
+  border: none;
+}
+
+.name {
+  width: 20vw;
+  height: 40vh;
+  margin-top: 2vh;
+}
```

Comment:
- [ ] When I tried to add a score, I couldn't see it on the list. Please fix it and also please add a scrolling behavior to your container as the list gets bigger it stretches to the bottom like this and seems unprofessional:
![image](https://github.com/Developer-Kingz/leader-board/assets/91486791/a66a3b56-45a3-4ecb-acd3-0acd38e14a85)